### PR TITLE
Using const() defines constants together (part:1)

### PR DESCRIPTION
**Pull request by xichengliudui**
Signed-off-by: xichengliudui <liuduidui@beyondcent.com>

### CODE REVIEW

**Comment by xichengliudui** [comment]
Divided into two parts.

**Comment by xichengliudui** [comment]
There should be no error in this modification, right? @sougou

**Comment by xichengliudui** [comment]
part:2: https://github.com/vitessio/vitess/pull/4820

**Comment by sougou** [comment]
Most of these constants should not be moved. It's because they're defined where they're used, and it's more readable that way. Could you rework this PR to only fix the places where the move makes things more readable?

**Comment by xichengliudui** [comment]
YES

**Comment by xichengliudui** [comment]
Can you specify which go files need constant blocks?

**Comment by xichengliudui** [comment]
Which files do not need to be modified? I'm going to update this pr

File: go/vt/vttablet/tabletconntest/fakequeryservice.go [hunk 1]
```diff
@@ -34,6 +34,37 @@ import (
 	vtrpcpb "vitess.io/vitess/go/vt/proto/vtrpc"
 )
 
+const (
```
Comment (sougou) [inline go/vt/vttablet/tabletconntest/fakequeryservice.go]: All these constants are used in the function immediately following their definition, which makes it more readable. So, I believe these should be moved back to where they were before.

Exception: if you see a constant used in more than one place, those can stay here.

File: go/vt/vttablet/tabletmanager/vreplication/table_plan_builder.go [hunk 2]
```diff
@@ -28,6 +28,21 @@ import (
 	querypb "vitess.io/vitess/go/vt/proto/query"
 )
 
+const (
```
Comment (sougou) [inline go/vt/vttablet/tabletmanager/vreplication/table_plan_builder.go]: These constanst should not be moved. They are defined right after their type, which makes it more readable.

File: go/vt/workflow/node.go [hunk 3]
```diff
@@ -42,6 +42,15 @@ import (
 type NodeDisplay int
 
 const (
+	// ActionStateUnknown is an unknown value and should never be set.
```
Comment (sougou) [inline go/vt/workflow/node.go]: These constants should be moved back (after `ActionState` definition.

**Pull request merged**